### ISSUE TITLE
Breaking change for systems < NW740: access return value's field in loop

### ISSUE THREAD

**Issue by GiantCrocodile** [larshp/abapGit#3454]
Hello,

I'm not able to activate abapGit in a NW 731 system because the source code contains this line 38092:

```
LOOP AT lo_dot->get_data( )-ignore INTO lv_ignore.
      lo_dot->remove_ignore( iv_path = ''
                             iv_filename = lv_ignore ).
    ENDLOOP.
```

This syntax was introduced in NW 740+.

Best Regards,

**Comment by larshp** [comment]
thanks for the bug report, pull requests welcome

**Comment by GiantCrocodile** [comment]
It would be good if someone can do a PR for this. I just saw you started to work on this @g-back. I will try to have a look on this with a colleague next week but I can't promise we are able to provide a good and working solution. If we do, we will create a PR for this of course.

**Comment by g-back** [comment]
I might as well try, looks simple enough to fix

**Comment by GiantCrocodile** [comment]
I was able to activate/compile abapGit in NW 731 now. Thank you for the fix. After executing the abapGit report, I got a dialog which asked me for a developer key for an object in SAP namespace. After cancelling that request and activating abapGit a second time, executing was able without this developer key request. I'm not sure why it asked me for that. If this happens again, I will check it in detail and might raise a new issue here.

Thank you for your support!
